ha/alert/node_alert_monitor.py

```diff
@@ -32,4 +33,39 @@ def process_alert(self):
         Log.debug("Processing event for NodeAlertMonitor")
         # Environment variable are avilable in self.crm_env
         self.iem = IemGenerator()
-        self.iem.generate_iem(self.crm_env["CRM_alert_node"], self.alert_event_module, self.alert_event_type)
+        alert_dup = AlertDuplication()
+        self.node_ids, self.local_node = alert_dup.AlertDuplication().get_status()
+
+        if self.node_ids[-1] == self.local_node:
```

Comment:
@shindeam Will this create any timing issue where the IEM will be missed from all nodes?